eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py

```diff
@@ -70,218 +69,213 @@ def create_table(private_config_xml_file):
+    query = f"""select taxonomy, scientific_name, assembly_accession, sources, fasta_path, report_path, 
+                release_folder_name from eva_progress_tracker.clustering_release_tracker 
+                where release_version = {curr_release_version - 1} and assembly_accession != 'Unmapped'"""
     with get_metadata_connection_handle("production_processing", private_config_xml_file) as pg_conn:
-        for tax, sc_name, asm_acc, src, fs_path, rpt_path, rls_folder_name in get_all_results_for_query(pg_conn, query):
-            if src == 'DBSNP - filesystem':
-                should_be_rls = False
-            else:
-                should_be_rls = True
-            tmp_instance = get_tempmongo_instance(pg_conn, tax)
-            query_insert = f"""INSERT INTO eva_progress_tracker.clustering_release_tracker( 
-                taxonomy,scientific_name,assembly_accession,release_version,sources,fasta_path,
-                report_path,tempmongo_instance,should_be_released, release_folder_name) 
-                VALUES ({tax},'{sc_name}','{asm_acc}',{curr_release_version},'{src}','{fs_path}',
-                '{rpt_path}','{tmp_instance}','{should_be_rls}', '{rls_folder_name}') ON CONFLICT DO NOTHING"""
-
-            execute_query(pg_conn, query_insert)
-
-
-def fill_num_rs_id_for_taxonomy_and_assembly(mongo_source, private_config_xml_file, release_version, taxonomy,
-                                             reference_directory):
-    assembly_list = get_assembly_list_for_taxonomy(private_config_xml_file, taxonomy, release_version)
-    pipeline = get_filter_group_count_pipeline(taxonomy, assembly_list)
-    cve_res = get_rs_count_per_assembly_in_collection(mongo_source, 'clusteredVariantEntity', pipeline)
-    dbsnp_res = get_rs_count_per_assembly_in_collection(mongo_source, 'dbsnpClusteredVariantEntity', pipeline)
+        with get_metadata_connection_handle(profile, private_config_xml_file) as pg_conn_insert:
+            for tax, sc_name, asm_acc, src, fs_path, rpt_path, rls_folder_name in get_all_results_for_query(pg_conn,
+                                                                                                            query):
+                tmp_instance = get_tempmongo_instance(pg_conn, tax)
+                query_insert = f"""INSERT INTO eva_progress_tracker.clustering_release_tracker(
+                                    taxonomy, scientific_name, assembly_accession, release_version, sources,
+                                    fasta_path, report_path, tempmongo_instance, release_folder_name) 
+                                    VALUES ({tax}, '{sc_name}', '{asm_acc}', {curr_release_version}, '{src}', 
+                                    '{fs_path}', '{rpt_path}', '{tmp_instance}', '{rls_folder_name}') 
+                                    ON CONFLICT DO NOTHING"""
+
+                execute_query(pg_conn_insert, query_insert)
```

Comment:
I would use `insert_entry_for_taxonomy_and_assembly ` allowing to pass parameters for  `sources`, `fasta_path`, `report_path`, etc

Reply:
done